presto-spi/src/main/java/com/facebook/presto/spi/plan/ValuesNode.java

```diff
@@ -37,13 +37,15 @@
 {
     private final List<VariableReferenceExpression> outputVariables;
     private final List<List<RowExpression>> rows;
+    private final Optional<String> valuesNodeLabel;
 
     @JsonCreator
     public ValuesNode(
```

Comment:
Can we create a new constructor which takes the label name as empty? This will limit the number of overall changes.

Reply:
> Can we create a new constructor which takes the label name as empty? This will limit the number of overall changes.

There could be issues with the json constructor. I think it expects only one. I had to go through hoops to make it work last time when I introduced the source location field in plan node.